kuma/wiki/helpers.py

```diff
@@ -324,8 +325,12 @@ def wiki_url(context, path):
     Create a URL pointing to Kuma.
     Look for a wiki page in the current locale, or default to given path
     """
+    request = context['request']
+    if waffle.flag_is_active(request, 'dumb_wiki_urls'):
```

Comment:
Right - it just skips all the "smart" logic bellow here. Per New Relic, the following calls to `memcache.get`, `reverse`, and `Document.objects`, and/or `redirect_url` make up as much as 5-10% of the time spent in `document.html`